cmd/podman/export.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	exportCommand.SetHelpTemplate(HelpTemplate())
 	exportCommand.SetUsageTemplate(UsageTemplate())
 	flags := exportCommand.Flags()
-	flags.StringVarP(&exportCommand.Output, "output", "o", "/dev/stdout", "Write to a file, default is STDOUT")
+	flags.StringVarP(&exportCommand.Output, "output", "o", "/dev/stdout", "Write to a file")
```

Comment:
For this and other STDOUT/STDIN things - should we say "instead of the terminal" to make it clear the default is the terminal?

Reply:
I think that makes it clearer, and am happy to change it, but would like confirming (or negating) opinions

Reply:
The key problem here is we have to deal with Cobra's automatic addition of "(default)":
```
  -o, --output string   Write output to FILE instead of stdout (default "/dev/stdout")
```
(And no, I can't figure out any way to tell Cobra to say "--output FILE" instead of "--output string"; but I haven't looked closely enough).

The only way to get rid of Cobra's "(default XXX)" is to make the default null. As you can see from the discussion in `start.go` (with `--sig-proxy`), that results in great uncertainty. I'm willing to try it, but before doing so, is there any way we can come up with a simple help message that looks good with the Cobra default?
```
  -o, --output string   INSERT HELPFUL MESSAGE HERE (default "/dev/stdout")
```
(IOW: we have control over HELPFUL MESSAGE; less so about the appended default).